packages/vue-components/src/panels/PanelBase.js

```diff
@@ -53,6 +53,22 @@ export default {
       type: Boolean,
       default: false,
     },
+    showPreview: {
+      type: Boolean,
+      default: false,
+    },
+    previewCardExpanded: {
```

Comment:
hmm are these props meant to be configurable?
if not, let's use `data()` per vue's recommendations.